src/test/miniminer_tests.cpp

```diff
@@ -309,6 +310,64 @@ BOOST_FIXTURE_TEST_CASE(miniminer_1p1c, TestChain100Setup)
             }
         }
     }
+
+    // Check m_inclusion_order for equivalent mempool- and manually-constructed MiniMiners.
+    // (We cannot check bump fees in manually-constructed MiniMiners because it doesn't know what
+    // outpoints are requested).
```

Comment:
Do we not know any outpoints or do we just not know whether the ancestor transactions of the submitted set are confirmed?

Reply:
It's just "I can't give an answer if I don't know what the question is." When using this constructor, the user is not specifying which outpoints they want bump fees for.